app/src/androidTest/java/com/sdpteam/connectout/profile/ProfileTest.java

```diff
@@ -143,6 +143,5 @@ public void deleteProfileTest() {
         fJoin(model.saveProfile(userProfile));
         model.deleteProfile(uid);
         waitABit();
-        assertNull(fJoin(model.fetchProfile(uid)));
```

Comment:
Same here, I do not understand why this assertion was removed

Reply:
because the fjoin will never return something since the uid doesn't even exist

Reply:
previously, it returns null, and then we don't render on the view depending on whether the profile is null or not. However, we don't even want the null profiles to be in the datasource

Reply:
This is much clearer now, thanks.